### ISSUE TITLE
Caching for ldap backend

### ISSUE THREAD

**Issue by secure-xxx** [glauth/glauth#428]
Hello! You doing cool things, respect!
Can you create caching mode for ldap backend?

**Comment by Fusion** [comment]
It depends. Could you be more specific?
Like, use glauth the cache the results retrieved from a LDAP backend?

**Comment by secure-xxx** [comment]
> It depends. Could you be more specific? Like, use glauth the cache the results retrieved from a LDAP backend?

No, i mean incoming byte caching without schema and etc. I test a little stand, imho it works
